src/styles/style.scss

```
@@ -0,0 +1,15 @@
+@import "utils/variables";
+@import "utils/mixins";
+
+@import "block/stars.scss";
+@import "block/card.scss";
+
+html {
+  font-family: "Roboto", sans-serif;
```

```suggestion
  font-family: "Roboto", Helvetica, sans-serif;
```
Checklist:
2. [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)